java-network/src/main/scala/com/linkedin/norbert/javacompat/network/BaseNettyNetworkClient.scala

```diff
@@ -17,14 +17,16 @@ package com.linkedin.norbert
 package javacompat
 package network
 
-import cluster.{Node, BaseClusterClient}
+import com.linkedin.norbert.EndpointConversions._
```

Comment:
we prefer not to use ._ (or .* in java) for imports, I know you didn't add this but it would be nice to explicitly list the imports